spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1264,6 +1266,23 @@ public void testRewriteJobOrderFilesDesc() {
     Assert.assertNotEquals("Number of files order should not be ascending", actual, expected);
   }
 
+  @Test
+  public void testPartitioningSortWithManualSortOnSameColumn() {
+    Table table = createTablePartitioned(4, 2);
+    table.refresh();
```

Comment:
Does this need to be called? The table instance returned by `createTablePartitioned` should be up to date.

Reply:
It shouldn't be required. I have removed it.